In [7]:
from math import factorial
import pandas as pd

In [2]:
factorial(3)

6

En un aeropuerto se tiene a 10 pasajeros esperando en la sala de preembarque, la polícia debe controlar a 3 de ellos. ¿Cuantas combinaciones posibles se pueden obtener?.

In [3]:
def cont_comb(total,muestra):
    return factorial(total)/(factorial(muestra)*factorial(total-muestra))
    

In [6]:
cont_comb(10,3)

120.0

La Agencia Nacional de Seguridad Vial realizó una investigación para saber si los conductores de están usando sus cinturones de seguridad. Los datos muestrales fueron los siguientes:
Conductores que emplean el cinturón


In [23]:
region=pd.DataFrame({'si':(148,162,296,252,858),'no':(52,54,74,48,228)},index=['Noreste','Noroeste','sur', 'centro', 'total'])
region

si   no
Noreste   148   52
Noroeste  162   54
sur       296   74
centro    252   48
total     858  228

a) ¿Qué metodo cree que se utilizo para asignar probabilidades?.
    'corriente clasica'

b) Construya un cuadro similar, pero con la asignación de probabilidades.


In [40]:
prob=pd.DataFrame(columns=['si','no'], index=['Noreste','Noroeste','sur', 'centro', 'total'])
sum=region['si']+region['no']
prob['si']=region['si']/sum
prob['no']=region['no']/sum
prob

si        no
Noreste   0.740000  0.260000
Noroeste  0.750000  0.250000
sur       0.800000  0.200000
centro    0.840000  0.160000
total     0.790055  0.209945

c) ¿Cuál sería el mejor método pára estimar la probabilidad de que en Estados Unidos un conductor lleve puesto el cinturón?.
bayesiana

d) Un año antes, la probabilidad en Argentina de que un conductor llevara puesto el cinturón era 0.75. El director de ANSV, se esperaba que la probabilidad llegara a 0.78. ¿Estará satisfecho con los resultados del estudio? 
si

e) ¿Cuál es la probabilidad de que se use el cinturón en las distintas regiones del país?. 0.790055

¿En qué región se usa más el cinturón?(Utilizar misma tabla que el ejercicio anterior).
centro

Crear una funcion que permita calcular a probabilidad de los siguientes eventos en un baraja de 52 cartas.
Obtener una carta roja.
Obtener una carta negra.
Obtener una pica.
Obtener un trébol.
Obtener un corazón.
Obtener un diamante.

In [78]:
def proba_baraja():
    inpu=input()
    if inpu=='roja' or inpu=='negra':
        return 26/52
    if inpu=='pica' or inpu=='corazón' or inpu=='trebol' or inpu=='diamante':
        return 13/52

In [82]:
proba_baraja()

0.25

La probabilidad de que salga un 7 o un 8 al seleccionar una carta de una baraja de las 52 cartas que contiene el mazo.


In [83]:
4/52

0.07692307692307693

La probabilidad de tu país gane el mundial de fútbol.


Un experimento que tiene tres resultados es repetido 50 veces y se ve que E1 aparece 20 veces, E2 13 veces y E3 17 veces. Asigne probabilidades a los resultados.

In [84]:
print('e1',20/50)
print('e2',13/50)
print('e3',17/50)

e1 0.4
e2 0.26
e3 0.34


Si la probabilidad de que un cliente pague en efectivo (E) es 6/15, con tarjeta de crédito (TD) es 7/15 y con tarjeta de débito (TD) es 2/15. 

Hallar la probabilidad de que dos clientes sucesivos que pagan sus cuentas lo hagan:

a) el primero en efectivo y el segundo con tarjeta de crédito.     r=0.18666666666666668

b) Los dos clientes en efectivo         r=0.16000000000000003

In [85]:
(6/15)*(7/15)

0.18666666666666668

In [86]:
(6/15)**2

0.16000000000000003

La probabilidad de que un Henry repruebe el M1 de 0.8, de que apruebe M2 es 0.5 y de que repruebe el M3 es de 0.4.
(Los eventos no interfieren entre si)
Determinar la probabilidad de que:

a) Apruebe un módulo.

b) Repruebe las tres materias.
0.16000000000000003


In [88]:
0.8*0.5*0.4

0.16000000000000003